README.md

```diff
@@ -1,2 +1,10 @@
-# hw01
-Repository for homework 01
+# Abby Bergman ![image](https://github.com/abbygirlrose/hw01/blob/master/ab.png)
+
+## Background
+#### I am from Los Angeles, California and have spent four years living in Western Massachusetts. I have a sister and a cat. 
+
+## Education
+#### I am a MAPSS student concentrating in **Psychology**. My research interests center around *motivation*, *decision making*, and *goals*. I completed my undergraduate education in **Neuroscience** and **Exercise and Sports Studies** at [Smith College](http://www.smith.edu), where I also competed on the varsity swim team. 
+
+## Interests
+#### I am fairly obsessed with swimming and will be volunteer coaching here at U of C. I am a marathon swimmer and details of some of my swims are located [here](http://abbybergman/com). 
```

Comment:
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |       X       |            |
| **Coding strategy**         |           |       X       |            |
| **Presentation**            |           |        X     |            |
| **Achievement, creativity** |      X     |              |            |
| **Ease of access**          |       X    |              |            |

Remarks:

* Great use of markdown elements! Great picture, I struggled to do that!
* There is an error in the second link.
* It would have been clearer if the content of each section was in regular text and not in fourth level heading. It diminished the effect of the second level headings. There is no need to use a # to keep content in regular text.
* Nice use of emphasis like italics and bold text.